polaris.shopify.com/content/components/box.md

```diff
@@ -1,27 +1,39 @@
 ---
 title: Box
-description: Box is the most primitive layout component. Box has a set of padding options. Use it to render an individual item.
+description: Box is the most primitive layout component. It’s a way to access Polaris design tokens.
 category: Structure
 keywords:
   - layout
 status:
   value: Alpha
   message: This component is in development. There could be breaking changes made to it in a non-major release of Polaris. Please use with caution.
 examples:
-  - fileName: box-with-padding.tsx
-    title: Padding
+  - fileName: box-default.tsx
+    title: Default
+    description: >-
+      By default, box has no padding, borders, background, or shadows, and border radius is set to zero.
+  - fileName: box-with-color.tsx
+    title: Color
+    description: >-
+      Background color of box and text color inside a box can be adjusted using the Color tokens.
+  - fileName: box-with-border.tsx
+    title: Border
     description: >-
-      Spacing inside Box can be set with the padding properties. Padding options are provided using our spacing tokens. There are 5 different padding properties: padding, paddingLeft, paddingRight, paddingTop, and paddingBottom.
+      Border width can be adjusted using the Shape tokens, and a subset of tokens allows different types of border.
   - fileName: box-with-border-radius.tsx
     title: Border radius
     description: >-
-      Border radius can be set with the border radius properties. Border radius options are provided using our shape tokens. There are 5 different padding properties: borderRadius, borderRadiusLeft, borderRadiusRight, borderRadiusTop, and borderRadiusBottom.
-  - fileName: box-with-background.tsx
-    title: Background
+      Border radius can be adjusted using the Shape tokens.
```

Comment:
Sorry I know the Figma file didn't have the link either. But can we put a link to the "Shape" tokens? Sorry I know the Figma file didn't have the link either. 

https://polaris.shopify.com/tokens/shape

Reply:
Yes I think so. Probably we need to go back to all component pages and make sure we are doing things consistently.

Reply:
agreed